lib/network/rest/RESTClient.js

```diff
@@ -8,6 +8,13 @@ const constants = require('../../constants');
 const utils = require('./utils');
 const errors = require('../../errors');
 
+const HttpAgent = require('agentkeepalive');
+
+// rest server is configured to wait 60 seconds of inactivity on a
+// HTTP socket before closing it, so we might reuse it safely for a
+// little bit less time to avoid ECONNRESET issues
+const FREE_SOCKET_TIMEOUT = 55000;
```

Comment:
should we reuse https://github.com/scality/Arsenal/blob/development/7.4/lib/constants.js#L85 ?

Reply:
nice catch, didn't know it was here, but im more concern by the fact that this do not fix the issue for good

Reply:
Can we reuse the constant from constants? :)

Reply:
yes i will, but i put this on hold the time i investigate why this fix is not enought :)

Reply:
done